resources-generator/src/main/kotlin/dev/icerock/gradle/generator/android/AndroidImagesGenerator.kt

```diff
@@ -38,22 +44,47 @@ class AndroidImagesGenerator(
             files.map { key to it }
         }.forEach { (key, file) ->
             val scale = file.nameWithoutExtension.substringAfter("@").substringBefore("x")
-            val drawableDirName = "drawable-" + when (scale) {
-                "0.75" -> "ldpi"
-                "1" -> "mdpi"
-                "1.5" -> "hdpi"
-                "2" -> "xhdpi"
-                "3" -> "xxhdpi"
-                "4" -> "xxxhdpi"
+            val drawableDirName = "drawable" + when (scale) {
+                "0.75" -> "-ldpi"
+                "1" -> "-mdpi"
+                "1.5" -> "-hdpi"
+                "2" -> "-xhdpi"
+                "3" -> "-xxhdpi"
+                "4" -> "-xxxhdpi"
                 else -> {
-                    println("ignore $file - unknown scale ($scale)")
-                    return@forEach
+                    if (file.svg) {
+                        ""
+                    } else {
+                        logger.info("ignore $file - unknown scale ($scale)")
```

Comment:
i think ignoring should be visible for developer - maybe developer do something wrong and not expect ignoring.
please change here `info` to `warn` level